src/components/Dashboard/DashboardGrid.jsx

```diff
@@ -162,6 +162,13 @@ const DashboardGrid = ({
     [childrenArray]
   );
 
+  const [animationState, setAnimationState] = useState(false);
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Is the timeout necessary here? useEffect is ran after the first render and after every update.

Reply:
> Is the timeout necessary here? useEffect is ran after the first render and after every update.

Hi @tay1orjones ,

The element animation depends on the property "shouldAnimate". It is initially set to false and animation should not be effect the element.
I'll change the value of "shouldAnimate" property to true and it should animate,
But it doesn’t as all of this happens really fast with setState , and browser decides to render changes on the end of result, 
That's why i used the settimeout, this way we can achive this animation effect.

Reply:
Yes, the timeout works because it prevents the batching of setState component updates. A better way to achieve this will be to use `requestAnimationFrame` instead of `setTimeout`.

Additionally, with this as-is, we have lost configurability of the animation via the `isEditable` prop.

I will push an update to this branch to make clear these two suggestions.